### PR TITLE
Added owebl web framework owebl.0.1

### PR DESCRIPTION
**Pull request by eatonphil**

### CODE REVIEW

File: packages/owebl/owebl.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,15 @@
+opam-version: "1.2"
+name: "OWebl"
```
Comment (avsm) [inline packages/owebl/owebl.0.1/opam]: this needs to match the filename. Lowercase preferred.

File: packages/owebl/owebl.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+version: "0.1"
+maintainer: "Phil Eaton <me@eatonphil.com>"
+author: "Phil Eaton <me@eatonphil.com>"
+homepage: "http://meetowebl.com"
+bug-reports: "https://github.com/eatonphil/owebl/issues"
+license: "BSD"
+dev-repo: ""
```
Comment (avsm) [inline packages/owebl/owebl.0.1/opam]: dev-repo can be "https://github.com/eatonphil/owebl.git"

**Comment by eatonphil** [comment]
Ok, great! Thanks!

File: packages/owebl/owebl.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+homepage: "http://meetowebl.com"
+bug-reports: "https://github.com/eatonphil/owebl/issues"
+license: "BSD"
+dev-repo: "https://github.com/eatonphil/owebl.git"
+build: [
+    [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "owebl"]
+depends: ["ocamlfind" "str" "unix"]
```
Comment (avsm) [inline packages/owebl/owebl.0.1/opam]: The `depends` field is for OPAM packages, and not for `ocamlfind` ones (which are one level down -- an OPAM package can contain multiple `ocamlfind` packages).  Including them leads to:

```
Package is unavailable on this configuration, skipping:
The following dependencies couldn't be met:
  - owebl -> unix
Your request can't be satisfied:
  - No package named unix found.
```

You can fix this just by removing `str` and `unix` from the `depends:` field above.

**Comment by eatonphil** [comment]
Got it, thanks again!

**Comment by avsm** [comment]
All green. Thanks!

**Pull request merged**